### PR TITLE
Threshold changes and gorilla list added

### PR DESCRIPTION
**Pull request by jwidness**
Changed the code for generating the official list to add gorillas, and modify the threshold for entry.  Also made a few other minor changes.

I'm sure there are bugs somewhere since I couldn't test the code, sorry!

### CODE REVIEW

File: bin/generate-lists.rb [hunk 1]
```diff
@@ -30,6 +44,45 @@ def tags_for(db, zooniverse_id)
     ])
 end
 
+def mod_tags_for(db, zooniverse_id)
+  db['discussions'].aggregate([
+      {'$match': {'focus._id': zooniverse_id, 'comments.$.tags': { '$ne': [] }}},
+      {'$project': {_id: 0, comments: 1}},
+      {'$unwind': '$comments'},
+      {'$match': {'$comments.user_name': {'$in': moderators}}}
```
Comment (adammcmaster) [inline bin/generate-lists.rb]: There's a missing comma here

File: bin/generate-lists.rb [hunk 2]
```diff
@@ -55,26 +108,44 @@ def tags_for(db, zooniverse_id)
...
+      all_tags = tags_for(db, document['zooniverse_id']).collect{ |tag| tag['_id'] }
+      chimp_count = species_count(document, 'chimpanzee')
+      gorilla_count = species_count(document, 'gorilla')
+      primate_count = species_count(document, 'other-(primate)')
+      
+      if !mod_tags.include?('omit') and (check_list_for_string(all_tags, 'chimp') or (is_verified(document) and chimp_count >= 2 and primate_count <= 2 * chimp_count))
+       add_to_hash(aggregate_species_hash, document, 'chimpanzee')
+      end
+      
+      if gorilla_sites.include?(site_name) and !mod_tags.include?('omit') and (check_list_for_string(all_tags, 'gorilla') or (is_verified(document) and gorilla_count >= 2 and primate_count <= 2 * gorilla_count)))
```
Comment (adammcmaster) [inline bin/generate-lists.rb]: There's one too many closing brackets at the end of this line

**Comment by adammcmaster** [comment]
@jwidness I've commented on a couple of syntax errors. Other than those this all seems to run successfully, so let me know once those are fixed and I'll merge this.

**Comment by jwidness** [comment]
I fixed those two things -- any ideas why sort doesn't work?  (see #11)
Thanks so much!

**Comment by adammcmaster** [comment]
Cool, merged!

I haven't had a chance to look at #11 yet, but will see if I can figure it out.

**Comment by jwidness** [comment]
Thanks so much!

**Comment by jwidness** [comment]
The official list hasn't updated -- will it do it automatically at some point, or does it need to be checked on?
Thanks!

**Comment by jwidness** [comment]
The official list is still down, any chance you could take a look at it?
Thanks!

**Comment by adammcmaster** [comment]
I think I have a fix for this. I'm testing it at the moment.

**Comment by adammcmaster** [comment]
Hmm, ok that didn't work. I'm going to generate this from the version of the code from before this was merged until I can look into this in more detail.

**Comment by jwidness** [comment]
Thanks for looking into that -- sorry I didn't respond, you caught me on moving day.

Is there anything I can do to try to get the new version up and running?  With another site just opening up, it would be super helpful if we could get this working!
Thanks again!

**Comment by mimiarandjelovic** [comment]
Hi, we are blocked with our progress as long as the new script is not working. could someone help us please?

**Comment by jwidness** [comment]
Super duper thanks for getting the new version going!!  There are a few unexpected behaviors, some of which I understand, and some of which I don't.

First, there is a problem in the other primate code that is causing all the videos with only blank counters to end up in the other primate list.  Second, integer division is putting videos in the list that I didn't expect.  For example ACP000e85h is 12 blank, 1 other primate, and 2 small grey duiker, and ACP000eivs is 14 blank and 1 small grey duiker. 
I think the best solution to both these problems is to make the test strictly greater than rather than greater than or equal to.
So instead of line 146
`if document['state'] == 'complete' and is_verified(document) and count >= ((classification_count - blanks) / 2)`
it would be
`if document['state'] == 'complete' and is_verified(document) and count > ((classification_count - blanks) / 2)`

Third, there isn't a gorilla list.  I don't know exactly what the problem is, but I noticed that although I updated both config.coffee and index.html with the pull request, the change to index.html isn't actually reflected on https://www.chimpandsee.org/identified-chimps/, so maybe the config change didn't go through either?  Or maybe there are other changes that need to be made too?

Fourth, videos with the #omit tag aren't omitted from the chimp list.  See http://talk.chimpandsee.org/#/subjects/ACP00083gs for example.  I don't have any thoughts for the source of this issue right now.

Fifth, the sort call is only working on some of the videos.  This is the strangest bug -- I'm really not sure why some of the videos are sorted and some are not.  It'd be great to have them all sorted by ID, but even better would be to have them sorted by filename.  If we can get the ID sort to work properly, I could add the code to switch it over to filename.

Thanks again!

**Comment by adammcmaster** [comment]
Oops, I forgot to deploy index.html. That should be updated now.

Could you open separate issues for the other problems? It'll be easier to keep track of each of them separately.

**Comment by jwidness** [comment]
Great, the gorilla list is solved, I opened a couple new issues (  #12 and #13 ) , and added a new comment to the sort issue already open ( #11 ).
Thanks!

**Pull request merged**